File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
import queue

def solution(k, m, score):
    answer = 0
    q = queue.Queue()
    sorted_score = sorted(score, reverse=True)

    for i in range(len(score)):
        q.put(sorted_score[i])
    
    for i in range(len(score)//m):
        for j in range(m):
            j = q.get()
        answer += j*m
    return answer

File: 프로그래머스/1/135808. 과일 장수/과일 장수.java
import java.util.Arrays;

class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        
        Arrays.sort(score);

        int idx = score.length;
        while (true) {
            idx -= m;
            if (idx < 0) break;
            answer += score[idx] * m;
        }
        
        return answer;
    }
}